Docs/Whats_New.md

```diff
@@ -22,7 +22,7 @@ Released TBA
   was sometimes incorrectly disabled due to a Metal driver bug.
 - Work around problems with using explicit LoD with arrayed depth images
   on Apple Silicon.
-
+- Added support for `VK_EXT_shader_atomic_float` *(requires Metal 3.0)*.
```

Comment:
```suggestion

```

Oops. I can't merge because it looks like your dev pull from this repo is not current, and it's causing a conflict in `Whats_New.md`.

The simplest thing to do is for you to just remove the change I requested to `Whats_New.md`, and I'll add `VK_EXT_shader_atomic_float` into that document when I clean it up before the upcoming Vulkan SDK release.